tests/tests_geomstats/test_riemannian_mean_shift.py

```diff
@@ -69,16 +69,27 @@ def test_single_cluster_riemannian_mean_shift(self):
 
         self.assertAllClose(expected, result)
 
-    @geomstats.tests.np_and_autograd_only
-    def test_double_cluster_riemannian_mean_shift(self):
-        gs.random.seed(10)
-        number_of_samples = 20
-        sphere = Hypersphere(dim=2)
-        metric = HypersphereMetric(2)
-
-        cluster = sphere.random_von_mises_fisher(kappa=20, n_samples=number_of_samples)
+    @staticmethod
+    def _test_double_cluster_init(
```

Comment:
`_init_double_cluster` may be a better naming, as no testing is performed there.

Reply:
Done.